.github/workflows/update_gh_pages.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow updates GitHub Pages on the repo it is run from.
+
+name: Update GitHub Pages
+
+on:
+  workflow_dispatch:  # Manually trigger. Colon is required.
+
+permissions:
+  contents: write  # For checkout and push.
+
+jobs:
+  update-github-pages:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: master
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: NPM install
+      # Use CI so that we don't update dependencies in this step.
+      # TODO: remove --force flag from installing examples when examples dependencies are routinely tested
+      # Until then, don't block updating just because some dependencies are out of date.
+      run: |
+        npm ci
+        cd examples
+        npm ci --force
+        cd ..
```

Comment:
Presumably this works because `npm ci` runs the `postinstall` script, which runs `lerna boot`—which, as far as I can tell, runs `npm install` rather than `ci`.  Is `--force` getting passed through to the per-directory `install` invocations?  (I wouldn't expect it to even be passed to the `lerna boot` call, but maybe `lerna` is doing something clever?)

Reply:
Tbh I don't know why this works, when you point it out. The first time I tried it, without the force flag, it threw errors and failed the run due to incompatible dependencies in the angular example, so I just tried it with adding the flag without stopping to consider how lerna works, and it succeeded. 🤔

Reply:
Huh.  TIL.  SGTM.